client/src/modules/cohort/index.jsx

```diff
@@ -429,6 +429,10 @@ class Cohort extends PureComponent {
               />
               <div className="cohort__details">
                 <MembersBox
+                  inviteData={{
+                    name,
+                    url: `cohort/${cohortId}`
```

Comment:
you can use here our: `routeGenerator(Routes.COHORT, cohortId)` helper from `helpers.js file`